WordPressKit/PluginState.swift

```diff
@@ -84,8 +83,10 @@ public struct PluginState: Equatable, Codable {
             && lhs.automanaged == rhs.automanaged
             && lhs.url == rhs.url
     }
+    
```

Comment:
We should get rid of whitespace changes that aren't "remove the extra spaces/tabs from this line" unless they improve code readability. Let's fix that across all files in this PR before we land this?